### ISSUE TITLE
Base64

### ISSUE THREAD

**Issue by jidan70** [scionoftech/webp-converter#14]
How do i use base64 images if i don't want to save files please? 
I don't find any packages which seem to be able to do that.

**Comment by Lonniebiz** [comment]
I agree, and would also like to see support for [binary buffers](https://github.com/scionoftech/webp-converter/issues/23).

**Comment by abdatta** [comment]
+1 Same, would appreciate if this is added!

**Comment by valbofs** [comment]
Any updates? Did you find any answers/packages?

**Comment by scionoftech** [comment]
Base64 is supported: `str2webpstr()` (base64 → webp base64) and `buffer2webpbuffer()` (buffer → webp buffer). See the usage guide. Closing 🙏 — reopen if you need something more here.
